src/webhooks/discord/dto/types.ts

```diff
@@ -0,0 +1,5 @@
+export type CreatorFiles = {
+  avatar?: string;
+  banner?: string;
+  logo?: string;
+};
```

Comment:
type names in plural are an automatic disqualification from passing a PR review 💀